### ISSUE TITLE
python generate_fashion_datasets.py creates empty directories

### ISSUE THREAD

**Issue by AhmedHashish123** [cuiaiyu/dressing-in-order#15]
Hi, when I run python generate_fashion_datasets.py. Two empty directories called train and test are created. No images are added inside them. I changed the data root folder to the one I made. My folder contains these files:
- testM_lip
- trainM_lip
- fashion-annotation-test.csv
- fashion-annotation-train.csv
- fashion-pairs-test.csv
- fashion-pairs-train.csv
- standard_test_anns.text
- test.lst
- train.lst

I don't think I'm missing any files, so I don't know why generate_fashion_datasets.py doesn't add images to the two folders it creates.

**Comment by AhmedHashish123** [comment]
I figured it out, there was a problem with the names

**Comment by AhmedHashish123** [comment]
Even after I corrected the names, the folders "train" and "test" are empty

**Comment by AhmedHashish123** [comment]
> I think it is probably still file naming issue. Maybe you can set some break point in the python file to figure out which lines causing the issue?

In one iteration of the loop, I print "path_names" and "train_images"

"path_names" gives this output:
dressing-in-orderimg_highresMENDenimid_0000008001_2_side.jpg

"train_images" gives this array (I will only show the first 3 elements of the array):
['fashionWOMENSkirtsid0000062904_3back.jpg', 'fashionWOMENTees_Tanksid0000783805_7additional.jpg', 'fashionMENPantsid0000161101_1front.jpg']

The names are completely different, I believe the dataset being used is the wrong one. I downloaded the inshop dataset like you said in the readme but the names are completely different. That is why the test and train folders are empty; there are no matching names between "path_names" and "train_images". What could be the cause of this problem? Do I need to download a different dataset?

I downloaded the last dataset in this link:
https://drive.google.com/drive/folders/0B7EVK8r0v71pQ2FuZ0k0QnhBQnc?resourcekey=0-NWldFxSChFuCpK4nzAIGsg
